subprojects/packagefiles/mpdecimal/tests/meson.build

```diff
@@ -0,0 +1,24 @@
+runtest = executable('runtest', files('runtest.c', 'test.c'), dependencies: mpdec_dep)
+
+testdata = custom_target(
+  'testdata',
+  command: [
+    find_program('gettests.py'),
+    join_paths(meson.current_source_dir(), 'testdata_dist'),
+    join_paths(meson.current_build_dir(), 'testdata')],
+  # The output file is not used but custom_target() requires to specify one.
+  output: 'testdata.stamp',
+  build_always_stale: true)
```

Comment:
I don't like this. Always running a python program in a no-op build is wasteful. Why not just create the stamp file?

Reply:
Indeed. I was too focuses on matching the native build exactly.

Reply:
Maybe I'm missing something, but it doesn't look like the stamp file actually gets created by the script even after your change?